plugins/source/stripe/client/testing.go

```diff
@@ -45,32 +43,33 @@ func MockTestHelper(t *testing.T, table *schema.Table, opts TestOptions) {
 		}
 	}()
 
-	newTestExecutionClient := func(ctx context.Context, logger zerolog.Logger, spec specs.Source, opts source.Options) (schema.ClientMeta, error) {
-		var stSpec Spec
-		if err := spec.UnmarshalSpec(&stSpec); err != nil {
-			return nil, fmt.Errorf("failed to unmarshal stripe spec: %w", err)
-		}
+	sched := scheduler.NewScheduler(scheduler.WithLogger(logger))
+	spec := &Spec{}
```

Comment:
don't you need to set up `APIKey` as you should err out on the `Spec.Validate`?

Reply:
fixed